torch/csrc/distributed/autograd/rpc_messages/cleanup_autograd_context_req.cpp

```diff
@@ -0,0 +1,46 @@
+#include <torch/csrc/distributed/autograd/rpc_messages/cleanup_autograd_context_req.h>
+#include <torch/csrc/jit/pickle.h>
+
+namespace torch {
+namespace distributed {
+namespace autograd {
+
+CleanupAutogradContextReq::CleanupAutogradContextReq(int64_t context_id)
+    : context_id_(context_id){};
+
+int64_t CleanupAutogradContextReq::getContextId() {
+  return context_id_;
+}
+
+rpc::Message CleanupAutogradContextReq::toMessage() && {
+  std::vector<at::IValue> ivalues;
+  ivalues.emplace_back(context_id_);
+
+  // pickle
+  std::vector<torch::Tensor> tensorTable;
+  std::vector<char> payload =
+      jit::pickle(c10::ivalue::Tuple::create(std::move(ivalues)), &tensorTable);
```

Comment:
Why do we need to create a tuple here? We just have one IValue, can't we just pickle that?

Reply:
I was thinking of keeping it this way since we could potentially avoid a refactor if we decide to start sending additional data for this RPC.

Reply:
For now, let's avoid using a tuple since it would have some overhead. If later on we need this we can change this accordingly.